### ISSUE TITLE
Preloader number not center in rtl body direction

### ISSUE THREAD

**Issue by HASSANDL** [amcharts/amcharts4#2383]
Preloader number not center in rtl body direction.
https://codepen.io/hassandl/pen/GRpjWzN
```
body {
  direction: rtl;
}
```

**Comment by martynasma** [comment]
Thanks for the heads up.

This will be fixed in the next version. Will notify you here when it is released.

In the interim you can fix it with this workaround:

```
chart.preloader.label.textAlign = "middle";
```

**Comment by HASSANDL** [comment]
Very thanks.

**Comment by martynasma** [comment]
Fixed in [4.9.18](https://github.com/amcharts/amcharts4/releases/tag/4.9.18).

## [4.9.18] - 2020-04-28

### Added
- Heatmaps can now be logarithmic (set `logarithmic = true` in the heat rule config).

### Fixed
- Preloader label was misaligned when RTL was enabled.
- Double "hit" event invocation on `MapPolygon` fixed.
- Regression plugin: "processed" event was triggered twice on initial chart load.
- SliceGrouper plugin: if slice templats had any "hit" events set, `clickBehavior = "break"` was not working.
- IE was not displaying series in `XYChartScrollbar`. On IE now gray fill/stroke is used instead of desaturate filter.
- IE was not displaying series with filters. Remnoved filters if IE.
- Fixed issue with zero-value nodes drill-down `SankeyDiagram`.
- When animations were disabled through options, showing `XYSeries` and `FunnelSeries` after it was hidden was not working.
- It was not possible to add `MapImages` to `MapImageSeries` directly from the GeoJSON geodata.
- `maxWidth` set on Legend's label was being ignored.
- Layout was not being redrawn when `minGridDistance` was set after chart initialization.
- If `sprite.showTooltipOn = "hit"` and no animated theme was used, tooltip used to flicker at old position before shown in the correct one.
- EDGE was not showing `ColumnSeries` gradients if gradient was set on series, not on directly on column template.


[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
